ow_lander/CMakeLists.txt

```diff
@@ -1,5 +1,4 @@
-cmake_minimum_required(VERSION 2.8.3)
-
+cmake_minimum_required(VERSION 3.1.3)

```

Comment:
Should this be the min required version in all our packages now?

Reply:
We can if we want to .. I think on our computers everyone has version 3.10

Reply:
If catkin 3.1.3 or newer is standard with ROS Melodic, this seems the right choice.  But if some of our users may be using an older catkin, 3.1 is more restrictive if not actually "required".

Reply:
this is a cmake version, it is different than catkin. I think on Ubuntu 18.04 you get cmake 3.10 but don't quote me on it

Reply:
ROS melodic packages originally used cmake 2.8.3. Some of them (such as https://github.com/ros/ros_comm) have been updated to 3.0.2. noetic is also on 3.0.2. To keep things useable for people who haven't updated their computers, I think using either of these versions is okay. I don't see a need to go beyond 3.0.2 unless there's a specific feature we need.

Reply:
This version bump came from moveit setup_assistant. I will try to lower it back and see if it breaks anything

Reply:
addressed